java/src/jmri/jmrix/mqtt/Bundle.properties

```diff
@@ -46,27 +46,26 @@ NameMessageLastWill=Last will message
 
 TopicBase=
 
-TopicTurnoutSend=track/turnout/
-TopicTurnoutRcv=track/turnout/
-
-TopicSensorSend=track/sensor/
-TopicSensorRcv=track/sensor/
-
-TopicLightSend=track/light/
-TopicLightRcv=track/light/
-
-TopicReporter=track/reporter/
-TopicSignalHead=track/signalhead/
-TopicSignalMast=track/signalmast/
-TopicLightCommand=/command
-
-TopicThrottleSend=cab/$address/throttle
-TopicThrottleRcv=cab/$address/throttle
-TopicDirectionSend=cab/$address/direction
-TopicDirectionRcv=cab/$address/direction
-TopicFunctionSend=cab/$address/function/$function
-TopicFunctionRcv=cab/$address/function/$function
-TopicConsistSend=cab/$address/consist
+TopicTurnoutSend=track/turnout/{0}
```

Comment:
@bobjacobsen 
I'm a bit concerned about this. It seems to me that this changes the default topics for the users. But I might be wrong. Maybe I don't understand what this change is about. I just want to check that we don't do any unwanted changes.

Reply:
The $address and $function topic placeholders for system ids were what I put in my initial version of Throttle and Consist that went into release 5.5.4.  I've just recently noticed that the existing Sensor and Turnout objects in MQTT used the {0} placeholder for system ids.  

This PR changes Throttle and Consist to use the {0} for system ids for consistency.  As it was only rolled out in release 5.5.4, I doubt anyone has used it in their layout except for me.  

The PR also makes it explicit in the default topic configurations for Sensor and Turnout that {0} is used as a placeholder for system ids.  The code implicitly adds them if they are missing but there is only a passing mention of them in the documentation (which I had missed).

TLDR:  Change only affects something that was implemented in the previous test release and hasn't made it into production yet.